operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-add-tech-alliance.md

```diff
@@ -0,0 +1,20 @@
+---
+description: 50% Draft
+---
+
+# How to add new tech alliance to https://mattermost.com/partners/
+
+1. Ask tech alliance to provide the following information to partner-directory@mattermost.com:
+
+ - Partner Name
+ - Partner Contact Information
+ - Partner Description
+ - City, State/Province and Country
+ - Logo 
+
+2. Sales Ops reviews the information, and either contacts the partner for additional details or forwards the information to Marketing
```

Comment:
```suggestion
2. Sales Ops reviews the information, and either contacts the partner for additional details or forwards the information to Marketing: 
```

Reply:
Do all contacts get emailed or is it an alias email address for each team?

Reply:
I wasn't sure what you meant by

> Do all contacts get emailed or is it an alias email address for each team?

Reply:
You can disregard that - I misread the section and on the second review realizes that it's a list of the contacts in those teams.